client/app/components/composites/SearchBar/SearchBar.js

```diff
@@ -88,11 +95,16 @@ class SearchBar extends Component {
       document.body.removeChild(container);
     }
   }
+  handleResize() {
+    this.setState({ mobileMenu: false });
```

Comment:
Window resize just closes the dropdown menu since the layout could change to the desktop version etc.